eulerpi/core/result_manager.py

```diff
@@ -440,9 +440,25 @@ def load_slice_inference_results(
                 )
                 overall_params = np.concatenate((overall_params, params))
 
+        num_walkers = inference_information["num_walkers"]
+
         # thin and burn in
         return (
-            overall_params[num_burn_in_samples::thinning_factor, :],
-            overall_sim_results[num_burn_in_samples::thinning_factor, :],
-            overall_density_evals[num_burn_in_samples::thinning_factor],
+            overall_params[
```

Comment:
`thinning_factor * num_walkers` should be the step width, not the end index.
Use the empty end index like before: `start_after_burned::thinning_step_width`, notice the `::` instead of `:`.